packs/data/equipment.db/ring-of-torag.json

```diff
@@ -6,7 +6,7 @@
         "baseItem": null,
         "containerId": null,
         "description": {
-            "value": "<p>This simple golden ring has a large, red gemstone set into it that sparkles with an inner fire.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">1</span> Interact</p>\n<p><strong>Frequency</strong> once per round</p>\n<p><strong>Requirements</strong> You're taking persistent fire damage</p>\n<p><strong>Effect</strong> You call upon the ring to put out the flame and its magic attempts to bring the fire under control. This is especially effective assistance against persistent fire damage, so you gain an immediate flat check to remove the persistent fire damage, and the DC typically decreases from 15 to 10.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">R</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You're targeted by a fire effect</p>\n<p><strong>Effect</strong> The ring's red gemstone flashes with light, granting you resistance 5 against fire and a +1 status bonus on your AC or saving throw against the fire effect targeting you; these benefits end as soon as the fire effect is resolved.</p>"
+            "value": "<p>This simple golden ring has a large, red gemstone set into it that sparkles with an inner fire.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">1</span> Interact</p>\n<p><strong>Frequency</strong> once per round</p>\n<p><strong>Requirements</strong> You're taking persistent fire damage</p>\n<p><strong>Effect</strong> You call upon the ring to put out the flame and its magic attempts to bring the fire under control. This is especially effective assistance against persistent fire damage, so you gain an immediate flat check to remove the persistent fire damage, and the DC typically decreases from 15 to 10.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">R</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You're targeted by a fire effect</p>\n<p><strong>Effect</strong> The ring's red gemstone flashes with light, granting you resistance 5 against fire and a +1 status bonus on your AC or saving throw against the fire effect targeting you; these benefits end as soon as the fire effect is resolved.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Effect: Ring of Torag]{Effect: Ring of Torag}</p>"
```

Comment:
This should probably be changed to be a toggle instead of an effect, keeping with the idea of effects being for things with durations and toggles being for either instantaneous or undefined durations.